modules/KalturaSupport/resources/mw.ClosedCaptions.js

```diff
@@ -441,10 +442,24 @@
 					_this.captionURLs = captionsURLs;
 					// Done adding source issue callback
 					mw.log( 'mw.ClosedCaptions:: loadCaptionsURLsFromApi> total captions count: ' + captions.length );
-					callback( captions );
+					// Check if we need to sort captions array Alphabetically
+					if( _this.embedPlayer.getKalturaConfig( "closedCaptions", "sortCaptionsAlphabetically" )) {
+						captions = _this.sortByKey( captions, 'language' );
+						callback( captions );
+					}
+					else {
+						callback( captions );
```

Comment:
Instead of `if(){}else{}` just use `if`, and always call `callback( captions );`